### PR TITLE
Setting Post title/content in constructor - #156

### PR DESCRIPTION
**Pull request by brucealdridge**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #156

### CODE REVIEW

**Comment by brucealdridge** [comment]
NB: fixes #156 and also resolves another issue where the fixture doesn't create title/content for Posts

File: src/Blog/Entity/Post.php [hunk 1]
```diff
@@ -96,8 +96,10 @@ class Post
      */
     private $comments;
 
-    public function __construct()
+    public function __construct(?string $title = null, ?string $content = null)
```
Comment (rustamwin) [inline src/Blog/Entity/Post.php:99]:
```suggestion
    public function __construct(string $title = '', string $content = '')
```

File: src/Blog/Entity/Post.php [hunk 2]
```diff
@@ -96,8 +96,10 @@ class Post
...
     {
+        $this->title = $title ?? '';
+        $this->content = $content ?? '';
```
Comment (rustamwin) [inline src/Blog/Entity/Post.php:102]:
```suggestion
        $this->title = $title;
        $this->content = $content;
```

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks!

**Pull request merged**